test/testcafe/spec/EnrollAuthenticatorPasswordView_spec.js

```diff
@@ -59,7 +61,6 @@ test.requestHooks(successMock)('should have both password and confirmPassword fi
   await enrollPasswordPage.waitForErrorBox();
   await t.expect(enrollPasswordPage.hasPasswordError()).eql(false);
   await t.expect(enrollPasswordPage.getConfirmPasswordError()).eql('New passwords must match');
-  await t.expect(enrollPasswordPage.getErrorBoxText()).eql('We found some errors. Please review the form and make corrections.');
```

Comment:
This was a redundant assertion.
`await enrollPasswordPage.waitForErrorBox();` does the same check